packages/block-library/src/heading/deprecated.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	getColorClassName,
+	RichText,
+} from '@wordpress/block-editor';
+
+const blockSupports = {
+	className: false,
+	anchor: true,
+};
+
+const blockAttributes = {
+	align: {
```

Comment:
Shouldn't this be `textAlign`, rather than `align`?

Reply:
The thing is that it is `align` as of today and this deprecation is used to replace it with `textAlign` in addition to using a class name in place of style.

Reply:
Oh, whoops. I didn't realize I was looking at `deprecated.js`.

Reply:
Unfortunately, we need to keep `align` as the name of this attribute for backward compatibility reasons :(